Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
```

Comment:
> Any suggestion ? Note that it is a SystemInfoCollector, not an InfoCollector.

Coming back to this _(sorry for late reply)_, I see that as well. I think I reacted since it was referred to during daly or something as infoCollector. As long as we refer to it by full name I'm fine with it.